src/code.sh

```
@@ -497,6 +497,19 @@ _upload_final_output() {
     dx-jobutil-add-output "metricsOutput" "$metrics_file_id" --class=file
     mv /home/dnanexus/out/gather/Results/MetricsOutput.tsv /tmp
 
+    # if user selects to not use the intermediate BAMs and fastqs,
+    # delete these files from the scatter directory
+    if [[ "$upload_intermediate_files" == false ]] ; then
```

Shellcheck (0.10.0)

[warning] upload_intermediate_files is referenced but not assigned.

(SC2154)